features/steps/partcad-cli/commands/status.py

```
@@ -0,0 +1,17 @@
+import os
+import yaml
+from behave import given, when, then
+
+
+@then('a file named "{filename}" should be created with content')
+def step_impl(context, filename):
+    expected_content = yaml.safe_load(context.text.strip())
+    location = os.path.join(context.test_dir, filename)
+    assert os.path.exists(location), f"File {location} does not exist"
```

Ruff (0.8.0)

Use of `assert` detected

(S101)